--- caja-dropbox-0.7.1/dropbox.in~	2012-12-23 20:27:53.000000000 +0200
+++ caja-dropbox-0.7.1/dropbox.in	2012-12-23 20:42:58.817057547 +0200
@@ -56,7 +56,7 @@
 
 PARENT_DIR = os.path.expanduser("~")
 DROPBOXD_PATH = "%s/.dropbox-dist/dropboxd" % PARENT_DIR
-DESKTOP_FILE = u"@DESKTOP_FILE_DIR@/dropbox.desktop"
+DESKTOP_FILE = u"@DESKTOP_FILE_DIR@/caja-dropbox.desktop"
 
 enc = locale.getpreferredencoding()
 
@@ -1255,7 +1255,7 @@
     # UBUNTU
     if u".config" in contents:
         autostart_dir = os.path.join(home_dir, u".config", u"autostart")
-        autostart_link = os.path.join(autostart_dir, u"dropbox.desktop")
+        autostart_link = os.path.join(autostart_dir, u"caja-dropbox.desktop")
         if should_autostart:
             if os.path.exists(DESKTOP_FILE):
                 if not os.path.exists(autostart_dir):
