--- caja-dropbox-1.7.90/./caja-dropbox.in~	2014-04-06 14:05:59.000000000 +0300
+++ caja-dropbox-1.7.90/./caja-dropbox.in	2014-04-06 14:09:55.077288285 +0300
@@ -50,7 +50,7 @@
 INFO = "Dropbox is the easiest way to share and store your files online. Want to learn more? Head to"
 LINK = "http://www.dropbox.com/"
 WARNING = "In order to use Dropbox, you must download the proprietary daemon."
-GPG_WARNING = "Note: python-gpg (python-gpgme for Ubuntu 16.10 and lower) is not installed, we will not be able to verify binary signatures."
+GPG_WARNING = "Note: python3-gpg is not installed, we will not be able to verify binary signatures."
 ERROR_CONNECTING = "Trouble connecting to Dropbox servers. Maybe your internet connection is down, or you need to set your http_proxy environment variable."
 ERROR_SIGNATURE = "Downloaded binary does not match Dropbox signature, aborting install."
 
